lib/notify/email.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Notify.Email do
+  @moduledoc """
+  This module holds the email notification functionality we use when updating users of various occurences in the system
+  """
+  import Bamboo.Email
+  import Bamboo.SendGridHelper
+
+  @doc """
+  This function builds an email that informs a user of their attempted changes being approved and applied
+  """
+  @spec change_approved(map()) :: Bamboo.Email.t()
+  def change_approved(pending_change) do
+    template_id = System.get_env("SENDGRID_APPROVED_TEMPLATE_ID")
```

Comment:
I believe we should prefer `Application.get_env/2` here

Reply:
Why? This value will not be checked into source control because it's a unique value to our Sendgrid account.

If we put in `Application` we'll loose the ability to change the ENV on-the-fly.